### DISCUSSION TITLE
对xquic新增拥塞控制算法后该如何编译

### DISCUSSION THREAD

**Discussion by TearsofLucifer** [alibaba/xquic#382] [Q&A] [answered]
如题，在xquic/src/congestion_control里添加其他拥塞控制算法后，在编译时需要修改哪些文件，可以编译生成更新后的libxquic.so

**Comment by derekwin** [comment, marked as answer]
你可以搜索cubic或者bbr关键词，然后在有这个关键字的对应位置仿照类似写法，加入你的算法相关的代码就行。

**Reply by TearsofLucifer** [reply]
嗯嗯，已经解决了，谢谢啦

**Reply by derekwin** [reply]
不客气
